ui/DesignSystem/Component/ProjectCard.svelte

```diff
@@ -50,7 +52,7 @@
         </div>
       {/if}
       {#if badge}
-        <Badge style="margin-left: 0.5rem" variant={badge} />
+        <Badge style="margin-left: 0.5rem" variant={BadgeType.Maintainer} />
```

Comment:
There's only one `BadgeType` right now; is there a reason to maintain the `variant` prop?